scripts/create_config.py

```diff
@@ -21,8 +21,23 @@
 # Access the lists using the argument names
 gmsl_pulsename = args.gmsl_pulse
 gmst_pulsename = args.gmst_pulse
-pulse_years = list(map(int, args.pulse_years))
-gases = args.gases
+
```

Comment:
@JMGilbert could just do a find-replace for CO2 -> CO2_Fossil in `gases` at this step and print a message saying so (so any output files w/ the "gas" in their name are not a surprise)

Reply:
I was considering this, but I'm nervous about inconsistencies in expectation for users who expect to see CO2 but find CO2_Fossil at various points in the pipeline

Reply:
ok we can leave it for now. I'll make a note in the README